.github/workflows/cuda.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         path: |
           ~/.ccache
           ~/.cache/ccache
-        key: ccache-cuda-nvcc-${{ hashFiles('.github/workflows/cuda.yml') }}-${{ hashFiles('cmake/dependencies/AMReX.cmake') }}
+        key: ccache-cuda-nvcc-${{ hashFiles('.github/workflows/cuda.yml') }}-${{ hashFiles('cmake/dependencies/AMReX.cmake') }}-${{ github.event.repository.pushed_at }}
         restore-keys: |
           ccache-cuda-nvcc-${{ hashFiles('.github/workflows/cuda.yml') }}-
```

Comment:
Lacks the `-${{ hashFiles('cmake/dependencies/AMReX.cmake') }}` key in restore.

Reply:
Adding `github.event.repository.pushed_at` to the restore keys is not necessary since it will always use the newest cache if multiple match the key.

Reply:
No, but the hashFile for `AMReX.cmake` is now missing